ntbs-service/Pages/Notifications/Edit/_TravelDetailsPartial.cshtml

```diff
@@ -12,7 +12,7 @@
             <div class="nhsuk-radios govuk-radios--conditional" data-module="govuk-radios" v-on:focusout="validate">
                 <div class="nhsuk-radios__item">
                     <input asp-for="TravelDetails.HasTravel" nhs-input-type="Radios" type="radio" value="true" 
-                           ref="hasDataYes" data-aria-controls="conditional-has-travel" />
+                           id="travelDetails-hasTravelYes" ref="hasDataYes" data-aria-controls="conditional-has-travel" />
                     <label asp-for="TravelDetails.HasTravel" nhs-label-type="Radios">
```

Comment:
ah, the custom ids also need to be added as `for`s on the labels sorry for not pointing out!
should be caught be running the WAVE accessibility checker though :)

Reply:
Absolutely right, I just assumed asp-for would link names rather than ids - WAVE (which I have now downloaded...) reported orphaned links which has now been remedied :)

Reply:
unforunately asp couldn't do that if it wanted to, as linking on ids is the web standard :)

Reply:
No way! You're absolutely right, I don't know where I possibly got that 'knowledge'...